### ISSUE TITLE
Abandoned modules, deceased authors, etc. (Raku Community Modules)

### ISSUE THREAD

**Issue by AlexDaniel** [Raku/problem-solving#168]
We have https://github.com/raku-community-modules org, but it's currently not defined what is supposed to happen with modules once they're there. Are these modules supposed to be transferred to those who are willing to maintain them, or are they supposed to be left in that org? And generally, shouldn't we have some policy or at least just some text somewhere defining what that org is and what is it for? Also, currently all people in the Raku org have access to all modules, is that OK or is it too dangerous?

IRC discussion: https://colabti.org/irclogger/irclogger_log/raku-dev?date=2020-03-12#l361

For inspiration: https://github.com/Perl-Toolchain-Gang/toolchain-site/blob/master/lancaster-consensus.md#flagging-abandoned-modules-and-modules-requesting-help

**Comment by sjn** [comment]
A useful minimum would be to add some instructions to the description of that group (Certainly something more actionable than "Module Adoption Center").

E.g.

"Modules found here are available for adoption. If you need one of these modules to have a safe and caring home, please sign up for maintainership by contacting <someplaceorthing>."

**Comment by sjn** [comment]
Oh my. This topic suddenly got more relevant than I had hoped. 😢

**Comment by AlexDaniel** [comment]
Yes, but it'll likely take a few weeks before we can talk about transferring @drforr's modules to @raku-community-modules. Also, it's not the first time something like this happens (but last time that is known to me we had a bit more time).
